pillar/cni.sls

```diff
@@ -17,6 +17,11 @@ flannel:
 # 8 - Display HTTP request contents.
   log_level:      '2'
 
+# the calico backend
+calico:
+  node: 'quay.io/calico/node:v2.6.9'
```

Comment:
I think we do not support images based on things other than `sles` (but @flavio should confirm this).

Reply:
It is fine as long, we keep this as empty pillar values